Deck.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Card_Game
+{
+    public abstract class Deck
+    {
+        protected List<Card> Cards { get; set; }
+        public bool IsEmpty()
+        {
+            if(Cards.Count < 1)
+            {
+                return true;
+            }
+            return false;
+        }
+        public void SortCards(List<Card> selectedCards, CardsAttributes attribute)
```

Comment:
Maybe the selectedCards parameter in this method (and another 2 below) could be removed and we would perform all those 3 methods on Cards field of the Deck?